src/pages/Login/Login.js

```diff
@@ -0,0 +1,62 @@
+
+import './login.scss';
+import { login } from '../../servises';
+
+
+export const Loader = () => {
+  return (
+    <div className="loader_wrap">
+      <div className="loader"></div>
+    </div>
+  );
+};
+
+export class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showLoader: false
+    };
+  }
+  submit = (e) => {
+    const { email, password } = e.target;
+    e.preventDefault();
+    this.setState({ showLoader: true });
```

Comment:
so, when loader will be hidden? Try to enter wrong login credentials and check what happens